### ISSUE TITLE
Saved Search Alert Creation Not Populating in Alerts page

### ISSUE THREAD

**Issue by jack-robs** [splunk/terraform-provider-splunk#79]
Hello!

I'm not seeing an available variable/field to set on the saved search docs https://github.com/splunk/terraform-provider-splunk/blob/master/docs/resources/saved_searches.md, such that alerts created via saved searches resource in TF populate in the `Alerts` page on Splunk, as they traditionally do when created through the UI.

Desired action: Create a splunk alert via `resource "splunk_saved_searches" "name"` such that it populates on the Splunk Search App: Alert page. 

Is there a way to do this or is there further context available? I understand that notionally with TF we're creating an alert which will work as intended, regardless of where it populates in the UI.

**Comment by jack-robs** [comment]
Hi friends, looks like it is fixed via a saved search resources needs `all` the possible fields added into the `main.tf`, IOT show up as an alert and on the Alerts page in Splunk vs. a report on the Saved Searches, Reports, and Alerts page.

**Comment by billycn20** [comment]
@jack-robs , i am facing the very same issue using the latest terraform Splunk provider. how did you resolve this? i didn't quite understand the solution you mentioned.

**Comment by billycn20** [comment]
updating this back for anyone else who's running into this problem, seems some additional parameters must be passed to terraform provider in order for it to appear as an alert, and not as a report. those additional parameters are:

```
    is_scheduled = "${var.is_scheduled}"
    cron_schedule = "${var.cron_schedule}"
    alert_track = "${var.alert_track}"
    alert_condition = "${var.alert_condition}"
```

**Comment by jack-robs** [comment]
Hey @billycn20 , will test out your fix. 

Apologies for the delay, but how we got it to work was including every possible field from this page into the saved search "object" that gets passed to terraform. We just set each of them to the default values listed. However, we didn't try to further prune how many fields we needed to include, so that's a great discovery if it's only those 4 that are required. 

https://github.com/splunk/terraform-provider-splunk/blob/master/docs/resources/saved_searches.md
